src/Tests/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs

```diff
@@ -508,6 +502,42 @@ public void ItCreatesCoverageFileInResultsDirectory()
             result.ExitCode.Should().Be(1);
         }
 
+        [PlatformSpecificFact(TestPlatforms.Windows)]
```

Comment:
please add test:

dotnet test --collect "Code Coverage;Format=Cobertura" --collect "XPlat Code Coverage"

it should generate 2 cobertura.xml files

Reply:
can we run also this test on linux?

Reply:
Demo test project doesn't contain coverlet reference so XPlat Code Coverage will not work. Though I added a test with two collect and it will verify that both collectors are passed.
Updated code coverage related tests to run on linux as well.